src/main/java/edu/ucla/library/bucketeer/Item.java

```diff
@@ -98,7 +104,7 @@ public IFilePathPrefix getFilePathPrefix() {
      * @throws IllegalArgumentException If the value from the CSV isn't a valid state
```

Comment:
Can the `@throws` part of the Javadoc be removed as well?

Reply:
Cf. the other related answer below.